tests/FunctionAwaitAnyTest.php

```diff
@@ -18,7 +18,7 @@ public function testAwaitAnyEmpty()
     public function testAwaitAnyFirstResolved()
     {
         $all = array(
-            $this->createPromiseRejected(1),
+            $this->createPromiseRejected(new \Exception(1)),
```

Comment:
Minor nit only: First argument is a `string` instead of an `int`. See also all following occurrences.

Reply:
Assuming you mean about the first argument of `Exception` taking a string? I've changed these to strings now.